bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -62,21 +61,13 @@ public WemoHandler(Thing thing, UpnpIOService upnpIOService, WemoHttpCall wemoHt
     @Override
     public void initialize() {
         super.initialize();
-        Configuration configuration = getConfig();
 
-        if (configuration.get(UDN) != null) {
```

Comment:
udn parameter can now never be null when initialize() is called ?

Reply:
No, this check is redundant in 3.3 as the parameter is defined as _required_ and we now have proper configuration validation: openhab/openhab-core#2691.